src/source/Sctp/Sctp.c

```diff
@@ -170,6 +170,38 @@ STATUS sctpSessionWriteMessage(PSctpSession pSctpSession, UINT32 streamId, BOOL
     return retStatus;
 }
 
+STATUS sctpSessionWriteDCEP(PSctpSession pSctpSession, UINT32 streamId, PCHAR pChannelName, UINT32 pChannelNameLen, PRtcDataChannelInit pRtcDataChannelInit)
+{
+     UNUSED_PARAM(pRtcDataChannelInit);
+     ENTERS();
+     STATUS retStatus = STATUS_SUCCESS;
+     struct sctp_sendv_spa spa;
+     PBYTE pPacket = NULL;
+     UINT32 pPacketSize = SCTP_DCEP_HEADER_LENGTH + pChannelNameLen;
+     UINT16 channelLenNetworkOrder = htons(pChannelNameLen);
```

Comment:
readIntXXX

Reply:
I am not able to find anything matching this, do you mind looking up the name again?
```
seaduboi@38f9d359441f:~/workspaces/AWSAcuityWebRTCClient/src/AWSAcuityWebRTCClient$ rg -i readint
open-source/amazon-kinesis-video-streams-producer-sdk-cpp/kinesis-video-pic/src/mkvgen/src/SpsParser.c
231:    INT32 readInt, width, height;
316:        CHK_STATUS(bitReaderReadExpGolombSe(&bitReader, &readInt));
319:        CHK_STATUS(bitReaderReadExpGolombSe(&bitReader, &readInt));
326:            CHK_STATUS(bitReaderReadExpGolombSe(&bitReader, &readInt));
```